hda-codec - Add Fujitsu Lifebook E8410 to quirk table

From: Tony Vroon <chainsaw@gentoo.org>

Add the proper model entry for Fujitsu Lifebook E8410 with ALC262 codec.
http://hg-mirror.alsa-project.org/alsa-kernel/rev/f86467a0bc3b

Index: linux-2.6.24-gentoo-r3/sound/pci/hda/patch_realtek.c
===================================================================
--- linux-2.6.24-gentoo-r3.orig/sound/pci/hda/patch_realtek.c
+++ linux-2.6.24-gentoo-r3/sound/pci/hda/patch_realtek.c
@@ -8510,6 +8510,7 @@ static struct snd_pci_quirk alc262_cfg_t
 	SND_PCI_QUIRK(0x103c, 0x2807, "HP D7000", ALC262_HP_BPC_D7000_WF),
 	SND_PCI_QUIRK(0x104d, 0x8203, "Sony UX-90", ALC262_HIPPO),
 	SND_PCI_QUIRK(0x10cf, 0x1397, "Fujitsu", ALC262_FUJITSU),
+	SND_PCI_QUIRK(0x10cf, 0x142d, "Fujitsu Lifebook E8410", ALC262_FUJITSU),
 	SND_PCI_QUIRK(0x17ff, 0x058f, "Benq Hippo", ALC262_HIPPO_1),
 	SND_PCI_QUIRK(0x17ff, 0x0560, "Benq ED8", ALC262_BENQ_ED8),
 	SND_PCI_QUIRK(0x17ff, 0x058d, "Benq T31-16", ALC262_BENQ_T31),
